.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+## I'm submitting a...
+<!-- Check one of the following options with "x" -->
+```
+[ ] Regression (a behavior that used to work and stopped working in a new release)
+[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
+[ ] Feature request <!-- Please file a UserVoice request and include the link below https://wpdev.uservoice.com/forums/110705-universal-windows-platform/category/193402-uwp-community-toolkit -->
+[ ] Sample app request
+[ ] Documentation issue or request
+[ ] Question of Support request => Please do not submit support request here, instead see https://github.com/Microsoft/UWPCommunityToolkit/blob/master/contributing.md#question
+```
+
+## UserVoice link
+<!-- If requesting new functionality -->
+
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+
+## Expected behavior
+<!-- Describe what the desired behavior would be. -->
+
+
+## Minimal reproduction of the problem with instructions
+<!--
+For bug reports please provide a *MINIMAL REPRO PROJECT* and the *STEPS TO REPRODUCE*
+-->
+
+
+## Environment
+<!-- For bug reports Check one or more of the following options with "x" -->
+```
+Nuget Package(s): 
+
+Package Version(s): 
+
+Windows 10 Build Number:
+- [ ] Anniversary Update (14393) 
+- [ ] Creators Update (15063)
+- [ ] Insider Build (xxxxx)
+
+App min and target version:
+- [ ] Anniversary Update (14393) 
+- [ ] Creators Update (15063)
+- [ ] Insider Build (xxxxx)
+
+Device form factor:
+- [ ] Desktop
+- [ ] Mobile
+- [ ] Xbox
+- [ ] Surface Hub
+- [ ] IoT
+
+Visual Studio 
+- [ ] 2017 15.3
+- [ ] 2017 15.4 Preview
```

Comment:
Remove 'preview'. It's implied today. And you'll have to change it once it ships

Reply:
Removed